src/services/transformEnums.service.js

```diff
@@ -9,38 +9,47 @@ const {
 
 // Functions that convert enums from keys to values.
 
-const transformEstablishmentTypeEnum = (establishmentType) => {
+const transformEstablishmentTypeEnum = (establishmentType, language) => {
+  const lang = language || "en";
```

Comment:
Could you not pass language = "en" as a default parameter to each of these functions?

Reply:
Good simplification, will do this thanks